app/styles/app.scss

```diff
@@ -219,7 +224,7 @@ $logo-dir: 'img/provider_logos/';
     }
 }
 .search-main {
-    background-color: #fbfbfb;
+    background-color: #fbfbfb !important;
```

Comment:
There is probably a better solution than `!important`

Reply:
Probably, but also probably not worth the effort atm 😆

Reply:
Fair, will hold off on this then